io/zenoh-transport/src/unicast/establishment/accept.rs

```diff
@@ -660,11 +675,10 @@ pub(crate) async fn accept_link(link: LinkUnicast, manager: &TransportManager) -
                 resolution: manager.config.resolution,
                 ext_qos: ext::qos::StateAccept::new(manager.config.unicast.is_qos),
                 #[cfg(feature = "transport_multilink")]
-                ext_mlink: manager
-                    .state
-                    .unicast
-                    .multilink
-                    .accept(manager.config.unicast.max_links > 1),
+                ext_mlink: match &manager.state.unicast.multilink {
```

Comment:
Should `ext_multilink` be an `Option` here as well? If that's the case, maybe `disabled()` is no longer required?

Reply:
Making it an `Option` would require also changing the `Cookie.ext_mlink` to `Option<ext::multilink::StateAccept>`, but then it's unclear to me how [this line](https://github.com/eclipse-zenoh/zenoh/blob/dev/1.0.0/io/zenoh-transport/src/unicast/establishment/cookie.rs#L64-L65) should be adapted.